src/traces/histogram/index.js

```diff
@@ -25,7 +23,8 @@ var histogram = module.exports = {};
  * constant and % work but they're not so meaningful. I guess it could be cool
  * to allow quadrature combination of errors in summed histograms...
  */
-Plotly.Plots.register(Plotly.Bars, 'histogram',
+
+Plotly.Plots.register(exports, 'histogram',
```

Comment:
... as discussed in https://github.com/plotly/plotly.js/issues/41#issuecomment-164554750